src/FontDescriptor.cc

```diff
@@ -0,0 +1,142 @@
+#include "FontDescriptor.h"
+
+using namespace Napi;
```

Comment:
I wrote this originally to help with the migration from NAN to NAPI since some of the types overlapped in names.
I generally don't like polluting the global namespace if it can be avoided so I'll just remove it.